### PR TITLE
style: Fix extraneous-parentheses (UP034)

### PR DESCRIPTION
**Pull request by echoix**
Ruff rule: https://docs.astral.sh/ruff/rules/extraneous-parentheses/

7 instances fixed

### CODE REVIEW

File: gui/wxpython/gmodeler/model.py [hunk 1]
```diff
@@ -3487,7 +3487,7 @@ def cleanup():
                 r"""    %s("g.remove", flags="f", type="vector",
                 name=%s)
 """
-                % (run_command, ",".join(('"' + x + '"' for x in vect)))
+                % (run_command, ",".join('"' + x + '"' for x in vect))
```
Comment (ninsbl) [inline gui/wxpython/gmodeler/model.py:3490]:
```suggestion
                % (run_command, ",".join(f'"{x}"' for x in vect))
```

File: python/grass/pygrass/modules/grid/grid.py [hunk 2]
```diff
@@ -346,8 +346,8 @@ def get_cmd(cmdd):
             if isinstance(vals, list)
         )
     )
-    cmd.extend(("-%s" % (flg) for flg in cmdd["flags"] if len(flg) == 1))
-    cmd.extend(("--%s" % (flg[0]) for flg in cmdd["flags"] if len(flg) > 1))
+    cmd.extend("-%s" % (flg) for flg in cmdd["flags"] if len(flg) == 1)
+    cmd.extend("--%s" % (flg[0]) for flg in cmdd["flags"] if len(flg) > 1)
```
Comment (ninsbl) [inline python/grass/pygrass/modules/grid/grid.py:350]:
```suggestion
    cmd.extend(f"-{flg}" for flg in cmdd["flags"] if len(flg) == 1)
    cmd.extend(f"--{flg[0]}" for flg in cmdd["flags"] if len(flg) > 1)
```

**Review by ninsbl — CHANGES_REQUESTED** [review]
We discussed scope of these changes earlier.
However, I would suggest to introduce f-strings in those small changes, even if not done all over the place.

**Comment by echoix** [comment]
> We discussed scope of these changes earlier.
> 
> However, I would suggest to introduce f-strings in those small changes, even if not done all over the place.

Would you be willing to attack all the rules that change to f-strings next, but limiting it to a certain amount changes per PR?

What length of PR would you like to have?

**Review by ninsbl — APPROVED** [review]

**Comment by echoix** [comment]
> > Would you be willing to attack all the rules that change to f-strings next, but limiting it to a certain amount changes per PR?
> > What length of PR would you like to have?
> 
> I am for sure willing to contribute to such an effort. Until now I mainly changed % string formating to f-string or .format() in files I edited for other reasons, so not systematically. What I found challenging is string management for SQL where one is not supposed to use string formating but rather parameters, ... I am just not sure what the best way of string handling is there and I do not want to change it to f-string / .format() just to later switch it to proper SQL handling at a later point... I would definately review PRs of managable size...

What's that size? Because it might get long if it was only in one. Do you have some reference on which recent PR was starting to be long?

**Pull request merged**